stl/inc/chrono

```diff
@@ -5965,8 +5990,20 @@ namespace chrono {
             case 'j':
                 if constexpr (_Is_specialization_v<_Ty, duration>) {
                     _Os << _STD abs(_CHRONO duration_cast<days>(_Val).count());
+                    return true;
                 }
-                return true;
+                if constexpr (is_same_v<_Ty, month_day_last>) {
+                    if (_Val.month() >= February) {
+                        _THROW(
+                            format_error("Cannot print the year day of the last day of any month other than January."));

```

Comment:
Not sure about "year day". Maybe "year-day" or "day of year"? Occurs below.

Reply:
"day of year" sounds best to me.